README.md

```diff
@@ -115,10 +118,10 @@ function findImports(path) {
   else return { error: 'File not found' };
 }
 
-// Current syntax
+// Old syntax (pre 0.5.12)
```

Comment:
I think we can't keep this here as people can't just copy paste the example anymore.